docs/testdata/TestGeneratePluginDocs-Markdown-relation_relation_table_a.md

```diff
@@ -4,7 +4,7 @@ This table shows data for Relation Relation Table A.
 
 Description for relational table's relation
 
-The primary key for this table is **_cq_id**.
+The composite primary key for this table is ().
```

Comment:
Based on this is it correct that tables without any primary keys will have this as a description? This shows up in our Website and might confuse users